### DISCUSSION TITLE
I met some problems in the process of using commonroad-scenario-designer, I would be grateful if anyone could help me!

### DISCUSSION THREAD

**Discussion by boxue-sun** [CommonRoad/commonroad-scenario-designer#25] [Q&A] [answered]
When I opened the downloaded folder like a tutorial and ran the start_gui.py file, I got an error like the one shown here. Somebody help me! (I was running in ubuntu20.04, anaconda and python3.8 virtual environment under vscode.)

![微信图片_20241119211722](https://github.com/user-attachments/assets/bbfbcdfa-f8e8-4c39-9174-d7af046925db)

**Comment by smaierhofer** [comment]
The Scenario Designer does not support Python 3.8. 
Please upgrade to see whether the error remains.

**Reply by boxue-sun** [reply]
Which version of Python would you recommend, please?

**Reply by smaierhofer** [reply]
Python 3.10 or Python 3.11

**Reply by boxue-sun** [reply]
Okay, I'll give it a try. Thank you.

**Reply by boxue-sun** [reply]
Hello, after I switched to version 3.10, I encountered this error again after running it. Please continue to help me solve it. Thank you
![微信图片_20241119211722](https://github.com/user-attachments/assets/38354798-9e4d-44cb-ad91-52621e3b35bc)

**Reply by smaierhofer** [reply]
Looks like the scenario designer is not correctly installed, the working directory is wrongly configured in the Python environment, or the script is executed from the wrong directory.

**Reply by boxue-sun** [reply]
Okay, I forgot to install the relevant libraries after switching to version 3.10. But after I installed the library, there was still an initial issue as shown in the picture when I ran it again.
![image](https://github.com/user-attachments/assets/d4c716b8-dc85-4276-bbdd-cbd2a939521f)

**Reply by smaierhofer** [reply, marked as answer]
Does this also happen if you execute `crdesigner` from the command line (just enter `crdesigner`)?
Maybe the wrong python environment is used (it looks like you have activated two in parallel: py3.10 and base). 

For me it works under Ubuntu with Python 3.10 and all our CI tests pass. 
Therefore, it is difficult to help further. 
I suggest to perform a google search on this error.

**Reply by boxue-sun** [reply]
ok thanks verymuch
